Author: Andreas Beckmann <anbe@debian.org>
Description: run clean target against the kernel version used for building
 the running kernel may not have the headers installed

--- a/dkms/dkms.conf
+++ b/dkms/dkms.conf
@@ -1,7 +1,7 @@
 PACKAGE_NAME="bbswitch"
 PACKAGE_VERSION="#MODULE_VERSION#"
 MAKE[0]="make KVERSION=$kernelver"
-CLEAN="make clean"
+CLEAN="make KVERSION=$kernelver clean"
 BUILT_MODULE_NAME[0]="bbswitch"
 DEST_MODULE_LOCATION[0]="/kernel/drivers/acpi"
 AUTOINSTALL="yes"
